commit de96d6b4bf7746a02a5663667691d73688f3cb20
Author: Vishesh Handa <me@vhanda.in>
Date:   Mon Nov 4 15:09:45 2013 +0100

    Nepomuk: DataManagement was changed to NepomukStorage
    
    Better error messages this way.
    nepomuk-core - e5f38c4fce42dfce982f2d184c2c5f433be7edd3

diff --git a/nepomuk/core/resourcedata.cpp b/nepomuk/core/resourcedata.cpp
index 9f09387..00cf6b6 100644
--- a/nepomuk/core/resourcedata.cpp
+++ b/nepomuk/core/resourcedata.cpp
@@ -300,7 +300,7 @@ bool Nepomuk::ResourceData::store()
         }
 
         QDBusConnection bus = QDBusConnection::sessionBus();
-        QDBusMessage msg = QDBusMessage::createMethodCall( QLatin1String("org.kde.nepomuk.DataManagement"),
+        QDBusMessage msg = QDBusMessage::createMethodCall( QLatin1String("org.kde.NepomukStorage"),
                                                            QLatin1String("/datamanagement"),
                                                            QLatin1String("org.kde.nepomuk.DataManagement"),
                                                            QLatin1String("createResource") );
@@ -486,7 +486,7 @@ void Nepomuk::ResourceData::setProperty( const QUrl& uri, const Nepomuk::Variant
 
         // update the store
         QDBusConnection bus = QDBusConnection::sessionBus();
-        QDBusMessage msg = QDBusMessage::createMethodCall( QLatin1String("org.kde.nepomuk.DataManagement"),
+        QDBusMessage msg = QDBusMessage::createMethodCall( QLatin1String("org.kde.NepomukStorage"),
                                                            QLatin1String("/datamanagement"),
                                                            QLatin1String("org.kde.nepomuk.DataManagement"),
                                                            QLatin1String("setProperty") );
@@ -540,7 +540,7 @@ void Nepomuk::ResourceData::addProperty( const QUrl& uri, const Nepomuk::Variant
 
         // update the store
         QDBusConnection bus = QDBusConnection::sessionBus();
-        QDBusMessage msg = QDBusMessage::createMethodCall( QLatin1String("org.kde.nepomuk.DataManagement"),
+        QDBusMessage msg = QDBusMessage::createMethodCall( QLatin1String("org.kde.NepomukStorage"),
                                                            QLatin1String("/datamanagement"),
                                                            QLatin1String("org.kde.nepomuk.DataManagement"),
                                                            QLatin1String("addProperty") );
@@ -589,7 +589,7 @@ void Nepomuk::ResourceData::removeProperty( const QUrl& uri )
         QMutexLocker lock(&m_modificationMutex);
 
         QDBusConnection bus = QDBusConnection::sessionBus();
-        QDBusMessage msg = QDBusMessage::createMethodCall( QLatin1String("org.kde.nepomuk.DataManagement"),
+        QDBusMessage msg = QDBusMessage::createMethodCall( QLatin1String("org.kde.NepomukStorage"),
                                                            QLatin1String("/datamanagement"),
                                                            QLatin1String("org.kde.nepomuk.DataManagement"),
                                                            QLatin1String("removeProperties") );
@@ -621,7 +621,7 @@ void Nepomuk::ResourceData::remove( bool recursive )
 
     if( !m_uri.isEmpty() ) {
         QDBusConnection bus = QDBusConnection::sessionBus();
-        QDBusMessage msg = QDBusMessage::createMethodCall( QLatin1String("org.kde.nepomuk.DataManagement"),
+        QDBusMessage msg = QDBusMessage::createMethodCall( QLatin1String("org.kde.NepomukStorage"),
                                                            QLatin1String("/datamanagement"),
                                                            QLatin1String("org.kde.nepomuk.DataManagement"),
                                                            QLatin1String("removeResources") );
diff --git a/nepomuk/core/resourcewatcher.cpp b/nepomuk/core/resourcewatcher.cpp
index 8a4906a..453be8f 100644
--- a/nepomuk/core/resourcewatcher.cpp
+++ b/nepomuk/core/resourcewatcher.cpp
@@ -68,7 +68,7 @@ Nepomuk::ResourceWatcher::ResourceWatcher(QObject* parent)
       d(new Private)
 {
     d->m_watchManagerInterface
-            = new org::kde::nepomuk::ResourceWatcher( "org.kde.nepomuk.DataManagement",
+            = new org::kde::nepomuk::ResourceWatcher( "org.kde.NepomukStorage",
                                                       "/resourcewatcher",
                                                       QDBusConnection::sessionBus() );
     d->m_connectionInterface = 0;
@@ -105,7 +105,7 @@ bool Nepomuk::ResourceWatcher::start()
     QDBusObjectPath path = reply.value();
 
     if(!path.path().isEmpty()) {
-        d->m_connectionInterface = new org::kde::nepomuk::ResourceWatcherConnection( "org.kde.nepomuk.DataManagement",
+        d->m_connectionInterface = new org::kde::nepomuk::ResourceWatcherConnection( "org.kde.NepomukStorage",
                                                                                      path.path(),
                                                                                      QDBusConnection::sessionBus() );
         connect( d->m_connectionInterface, SIGNAL(propertyAdded(QString,QString,QVariantList)),
